registry/infrastructure/models/models.py

```diff
@@ -138,3 +138,81 @@ class OrganizationAddressHistory(Base):
     country = Column("country", VARCHAR(64), nullable=False)
     created_on = Column("created_on", TIMESTAMP(timezone=False))
     updated_on = Column("updated_on", TIMESTAMP(timezone=False))
+
+
+# DB Model Class for Service
+
+class Service(Base):
+    __tablename__ = "service"
+    row_id = Column("row_id", Integer, primary_key=True, autoincrement=True)
+    uuid = Column("uuid", VARCHAR(128))
+    display_name = Column("display_name", VARCHAR(128), nullable=False)
+    service_id = Column("service_id", VARCHAR(128), nullable=False)
+    metadata_ipfs_hash = Column("metadata_ipfs_hash", VARCHAR(255))
+    proto = Column("proto", JSON, nullable=False)
+    short_description = Column("short_description", VARCHAR(1024), nullable=False)
+    description = Column("description", VARCHAR(1024), nullable=False)
+    git_url = Column("git_url", VARCHAR(512), nullable=False)
+    assets_hash = Column("assets_hash", JSON, nullable=False)
+    assets_url = Column("assets_url", JSON, nullable=False)
+    rating = Column("ratings", JSON, nullable=False)
+    contributors = Column("contributors", JSON, nullable=False)
+    created_on = Column("created_on", TIMESTAMP(timezone=False))
+    updated_on = Column("updated_on", TIMESTAMP(timezone=False))
+
+
+class ServiceReviewWorkflow(Base):
+    __tablename__ = "service_review_workflow"
+    row_id = Column("row_id", Integer, primary_key=True, autoincrement=True)
+    service_uuid = Column("service_uuid", Integer,
+                          ForeignKey("service.uuid", ondelete="CASCADE", onupdate="CASCADE"),
+                          nullable=False)
+    status = Column("status", VARCHAR(128), nullable=False)
+    transaction_hash = Column("transaction_hash", VARCHAR(128))
+    created_by = Column("created_by", VARCHAR(128), nullable=False)
+    updated_by = Column("updated_by", VARCHAR(128), nullable=False)
+    approved_by = Column("approved_by", VARCHAR(128))
+    created_on = Column("created_on", TIMESTAMP(timezone=False))
+    updated_on = Column("updated_on", TIMESTAMP(timezone=False))
+
+
+class ServiceGroup(Base):
+    __tablename__ = "service_group"
+    row_id = Column("row_id", Integer, primary_key=True, autoincrement=True)
+    service_uuid = Column("service_uuid", Integer,
+                          ForeignKey("service.uuid", ondelete="CASCADE", onupdate="CASCADE"),
+                          nullable=False)
+    group_id = Column("group_id", VARCHAR(128), nullable=False)
+    name = Column("name", VARCHAR(128), nullable=False)
```

Comment:
should not keep group name at org and service level both. just keep reference of group in service

Reply:
group name will be stored at the org group level.